scaling_performance/optimizing_storage.adoc

```diff
@@ -455,6 +455,11 @@ significant overlap in image content.
 Page cache sharing is not possible with DeviceMapper because thin-provisioned
 devices are allocated on a per-container basis.
 
+[NOTE]
+====
+OverlayFS is the default configuration for Red Hat Enterprise Linux (RHEL) 7.5 and is supported in 7.3 and later. OverlayFS is strongly recommended as the default Docker storage configuration on RHEL. See the link:https://access.redhat.com/solutions/2908851[instructions] for configuring OverlayFS for use with the docker container runtime.
```

Comment:
* "is the default configuration" or "is part of the default configuration for"? Or is it just installed by default?
* Checkout and "Claims and recommendations" in the IBM Style Guide if you want to see suggestions for removing the "is strongly recommended" construction.
* "docker" -> "Docker"